plugins/Jobs.h

```diff
@@ -11,6 +11,8 @@ class BedrockPlugin_Jobs : public BedrockPlugin {
     virtual bool processCommand(SQLite& db, BedrockCommand& command);
 
   private:
+    int64_t lastJobID;
```

Comment:
Heh, reviewing my own code... I thought that this is actually wrong as it's not thread safe, right? Should I use a `std::atomic_int`?

Reply:
Depends on how many threads can change it.